MdeModulePkg/Core/Pei/Dispatcher/Dispatcher.c

```diff
@@ -5,6 +5,8 @@ Copyright (c) 2006 - 2024, Intel Corporation. All rights reserved.<BR>
 (C) Copyright 2016 Hewlett Packard Enterprise Development LP<BR>
 SPDX-License-Identifier: BSD-2-Clause-Patent
 
+Copyright (c) 2017, Microsoft Corporation.

```

Comment:
Copyright (C) Microsoft Corporation.

Please ensure any additions have this format.